ui/components/multichain/asset-picker-amount/hooks/useIsFiatPrimary.tsx

```diff
@@ -0,0 +1,17 @@
+import { useSelector } from 'react-redux';
+import {
+  getPreferences,
+  getSendInputCurrencySwitched,
+} from '../../../../selectors';
+
+export default function useIsFiatPrimary() {
```

Comment:
can we write a test for this hook ?

Reply:
Replace this with a memoized selector, but I wrote a test for that in 597ebae